index.bs

```diff
@@ -31,26 +31,23 @@ Status Text: This WebXR Augmented Reality Module is designed as a module to be i
 <pre class="link-defaults">
 spec:infra;
     type:dfn; text:string
+spec:webxr-1;
```

Comment:
yess the bikeshed data updates finally happened!

Reply:
Yeah, meant to leave a comment about that. It was freaking out about linking to "immersive-vr" and "inline", and this was the recommended way to fix it. Turns out it ALSO let us get rid of a bunch of explicit links! Bonus!